From 14f0b6e8e9b6638c34853231df1e5eae53b24453 Mon Sep 17 00:00:00 2001
From: kfx <kfx@rock-chips.com>
Date: Sat, 28 Jul 2012 16:22:49 +0800
Subject: [PATCH] Update rk30 sdk adckey sample value(board_rk30_sdk_key.c)

rk3066 core board(default): //define RK3000_SDK
rk3000 core board: define RK3000_SDK
---
 arch/arm/mach-rk30/board-rk30-sdk-key.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-rk30/board-rk30-sdk-key.c b/arch/arm/mach-rk30/board-rk30-sdk-key.c
index f31366330ab7..e2904e322ff8 100755
--- a/arch/arm/mach-rk30/board-rk30-sdk-key.c
+++ b/arch/arm/mach-rk30/board-rk30-sdk-key.c
@@ -6,7 +6,7 @@
 
 #define PRESS_LEV_LOW			1
 #define PRESS_LEV_HIGH			0
-
+//#define RK3000_SDK   //rk3000 sdk core board
 static struct rk29_keys_button key_button[] = {
 	#if 0
 	{
@@ -78,7 +78,7 @@ static struct rk29_keys_button key_button[] = {
 		.active_low = PRESS_LEV_LOW,
 	},
 	#endif
-#ifdef CONFIG_MACH_RK3066_SDK
+#ifndef RK3000_SDK
 	{
 		.desc	= "menu",
 		.code	= EV_MENU,
-- 
2.35.3

